lib/formatters/stringFormatter.js

```diff
@@ -47,22 +47,23 @@ function deprecationsFormatter(results) {
 	}
 
 	const seenText = new Set();
+	const lines = [];
 
-	return allDeprecationWarnings.reduce((output, warning) => {
-		if (seenText.has(warning.text)) return output;
+	for (const { text, reference } of allDeprecationWarnings) {
+		if (seenText.has(text)) continue;
 
-		seenText.add(warning.text);
+		seenText.add(text);
 
-		output += yellow('Deprecation Warning: ');
-		output += warning.text;
+		let line = ` ${dim('-')} ${text}`;
 
-		if (warning.reference) {
-			output += dim(' See: ');
-			output += dim(underline(warning.reference));
+		if (reference) {
+			line += dim(` See: ${underline(reference)}`);
 		}
 
-		return `${output}\n`;
-	}, '\n');
+		lines.push(line);
+	}
+
+	return ['', yellow('Deprecation Warning:'), ...lines, ''].join('\n');
```

Comment:
```suggestion
	return ['', yellow('Deprecation warnings:'), ...lines, ''].join('\n');
```

Minor nit, otherwise LGTM to merge.

Reply:
Nice. Fixed via bec7cdc7c.